.github/workflows/create-release.yml

```diff
@@ -6,12 +6,6 @@ on:
       milestone_title:
         description: 'Milestone title (this release)'
         required: true
-  push:
-    branches:
-      - master
-  pull_request:
-    branches:
-      - '*'
```

Comment:
There is perhaps a more nuanced approach we can take that would allow us to run the create-release.yml workflow on PRs without using additional macOS runners. For example, we could choose to only run the workflow if `requirements.txt` changes. We could even compare the `requirements-freeze.txt` artifact to a cached version, and only run the test suite if this file has changed. But, given the care required to implement such a solution, and given the need to unblock PRs ASAP, I figure we could try this in a follow-up PR.

Reply:
I think it's fine if we only run the create-release workflow as a daily cron job (or on-demand, if it's in the days leading up to a release).